network_configs/addresses.env

```diff
@@ -0,0 +1,17 @@
+# Unique Coinbase addresses
+ZONE_0_0_COINBASE=0x0000000000000000000000000000000000000001
+ZONE_0_1_COINBASE=0x0100000000000000000000000000000000000001
```

Comment:
are these valid addresses ?

Reply:
They are default addresses. Go-quai was throwing an error when I initialized them all to 0x00000000 so I had to add a valid address in each shard. They are just a void, however. Need to figure out how we want to do this though. Once you get to a 4x4, there are 16 addresses that is pretty annoying for a user to have to fill in